src/Core/Addon/AddonsCollection.php

```diff
@@ -66,7 +66,7 @@ public function __construct(array $addons = [])
      */
     public static function createFrom(array $addons)
     {
-        return new static($addons);
+        return new self($addons);
```

Comment:
Always use static, this class can be extended

Reply:
Fixed ;)